sharething_pwa/src/AddItem/AddItemScreen.tsx

```diff
@@ -33,7 +40,10 @@ class AddItemScreen extends React.Component<Props, State> {
     }
 
     onSubmit = (event: FormEvent<HTMLFormElement>) => {
-        this.props.firebase.pushItem(this.state.itemName, this.state.itemDescription);
+        this.props.firebase.pushItem(this.state.itemName, this.state.itemDescription)
```

Comment:
this would be a good place to use object destructuring.
`const { itemName, itemDescription } = this.state;`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment